### DISCUSSION TITLE
Error in constructor :  class GCSToBigQueryOperator

### DISCUSSION THREAD

**Discussion by clementbecarelli** [apache/airflow#16713] [Ideas]
Hello, in this [file](https://github.com/apache/airflow/blob/4b14d93f25f237b8cca17f57352c253f292a111b/airflow/providers/google/cloud/transfers/gcs_to_bigquery.py#L29)

A piece of code seems to be missing

```
if self.external_table:
    [...]
     autodetect=self.autodetect,
    [...]
```
This must be an oversight but it prevents the creation of an external table with an automatically deduced schema.

Have a nice day.

**Comment by potiuk** [comment]
You need to fork the project @clementbecarelli and you can make a PR from it .

**Reply by potiuk** [reply]
You are most welcome to.
